packages/react-search-ui-views/src/Result.js

```diff
@@ -59,7 +59,7 @@ function Result({
 }) {
   const fields = getEscapedFields(result);
   const title = getEscapedField(result, titleField);
-  const url = getRaw(result, urlField);
+  const url = getUrlSanitizer(URL, location)(getRaw(result, urlField));
```

Comment:
Sorry, this is me being real slow in the morning, but why are we currying a native URL function here but an imported core-js URL function in our tests?

Reply:
Because URL is available in the browser, but not in the node environment that jest runs

Reply:
Ahhh - hm, is this not the browser equivalent of URL()? https://nodejs.org/api/url.html

Reply:
Oh wait, right, the node env specific to Jest 🤦‍♀️ 

I believe there is a specific flag or config you can use at the top of the test to tell Jest to run it as a node env (instead of jest's default jsdom env), but not sure if you want to go back and look at it now rather than focusing on getting this out and shipped.

After some thought I think my personal preference would have been mocking URL() in Jest rather than currying it (which for some reason just makes me a little nervous that we're potentially not in full control of the URL parser/is another possible vector for attack), but that's just me being paranoid and splitting hairs :)

Reply:
Oh I'm sorry, I didn't realize there was a URL class available in node. I wonder if it is consistent with the browser's API.

Reply:
Any specific thought around how it could be used as an attack vector? We could probably change this.

Reply:
Nah it totes can't, sorry for the unclear verbal diarrhea. What I meant was more that whenever I see something as an arg, mentally I tend to think "this is dynamic and changes" and also sometimes "the user is passing this in", vs. a regular import/constant which definitely reads to me as "this is defined by only us".

If we had infinite time I'd like to come back and maybe look at converting this to mocking URL() in jest instead of passing a curried function, but that can be tech debt for another day!

Reply:
Makes total sense. I'll still need to port this forward to master, so I could try to clean it up then.